blog/views.py

```diff
@@ -41,6 +42,9 @@ def article_create():
     if request.method == 'POST':
         article.title = request.form.get('title')
         article.content = request.form.get('content')
+        if article.title == '' or clean_html(article.content) == '':
+            return render_template('blog/article.html', article=article,
```

Comment:
A more conventional way to pass errors to a template is a variable called `errors` [like here](https://github.com/neomad-team/neomad.org/blob/master/auth/views.py#L28).

+1 for the _400_ status 👍